### DISCUSSION TITLE
absolute positioning not working on Osu Stable

### DISCUSSION THREAD

**Discussion by XReDoX** [ppy/osu#28470] [Q&A]
I wanted to report a problem that I still have, so when I play Osu Stable, absolute positioning doesn't work on my tablet, but when I run osu lazer! absolute positioning works normally. is there any way to fix it? If not, everything is fine, I like playing Osu Lazer
tablet: Huion h430p
(tablet drivers are downloaded)

**Comment by Stoppedpuma** [comment]
Issues related to stable should be posted [here](https://github.com/ppy/osu-stable-issues) and isn't even really an issue with stable, enabling or disabling raw input should work though.
